authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -1,9 +1,10 @@
-# Generated by Django 2.1.7 on 2019-03-19 12:02
+# Generated by Django 2.1.7 on 2019-03-30 08:47
```

Comment:
@Rhytah, ideally, migrations should not be edited unless you are making a change to the model and in which case a new file will be created